compiler/passes/scopeResolve.cpp

```diff
@@ -1787,9 +1787,6 @@ static void lookup(BaseAST*       scope,
 
     if (symbols.n == 0) {
       if (scope->getModule()->block == scope) {
-        ModuleSymbol* mod = scope->getModule();
-        lookup(mod->block, name, symbols, alreadyVisited);
-
         if (symbols.n == 0 && getScope(scope)) {
```

Comment:
Good point!  With the lookup call removed, (and the analysis that it wouldn't have affected symbols.n anyways) symbols.n is guaranteed to be the same value it was before.  I think I'll remove that portion of the line and retest (and will continue to work towards overhauling lookup)